h/browser/chrome/lib/sidebar-injector.js

```diff
@@ -3,6 +3,32 @@
 var blocklist = require('../../../static/scripts/blocklist');
 var errors = require('./errors');
 var settings = require('./settings');
+var util = require('./util');
+
+var CONTENT_TYPE_HTML = 'HTML';
+var CONTENT_TYPE_PDF = 'PDF';
+
+// a function which is executed as a content script
+// to determine the type of content being displayed in a tab
+function detectContentType(element) {
+  // check if this is the Chrome PDF viewer
+  var element = element || document;
```

Comment:
Is `element` actually used? Is it good to have both a param and a var named `element`?